nixos/tests/ihatemoney.nix

```diff
@@ -0,0 +1,49 @@
+{ system ? builtins.currentSystem,
+config ? {},
+pkgs ? import ../.. { inherit system config; }
+}:
+
+let
+  inherit (import ../lib/testing.nix { inherit system pkgs; }) makeTest;
+in map (backend: makeTest {
+  name = "ihatemoney-${backend}";
+  machine = { lib, ... }: {
+    services.ihatemoney = {
+      enable = true;
+      enablePublicProjectCreation = true;
+      inherit backend;
+      uwsgiConfig = {
+        http = ":8000";
+        };
+        };
+        boot.cleanTmpDir = true;
```

Comment:
Wanted to go ahead and merge, but there's just this remaining place with wrong indentation -- the below are just nits seen while re-reading through :)